lib/compute/index.js

```diff
@@ -50,17 +64,17 @@ exports.image = function image(id, payload) {
         return reject(err);
       }
       var buf = Buffer.concat(arr);
-      imageCc.enqueue({
-        id: id,
-        payload: buf
-      }, function(err, res) {
-        if (err) {
-          logger.error('process.error', err);
-          reject(AppError.processingError(err));
-        } else {
-          resolve(res);
-        }
-      });
+
+      resolve(P.all(Object.keys(SIZES).map(function(variant) {
+        var hw = SIZES[variant];
+        return enqueue({
+          id: id,
+          suffix: (variant === 'default') ? '' : variant,
+          height: hw.h,
+          width: hw.w,
+          payload: buf
+        });
+      })));
```

Comment:
Yeah, the more I think about all the edge cases here, the less I want to tackle it.  We can always spin off cleanup to a follow-up bug.